src/Uno.UI/UI/Xaml/Controls/Flyout/Flyout.cs

```diff
@@ -88,23 +86,22 @@ private void OnPopupOpened(object sender, object e)
 		{
 			var child = _popup.Child as FrameworkElement;
 
-			if(child != null)
-			{
-				SizeChangedEventHandler handler = (_, __) => SetPopupPositionPartial(Target);
-
-				child.SizeChanged += handler;
-
-				_sizeChangedDisposable.Disposable = Disposable
```

Comment:
Removing the disposable will create a memory leak. Why was it removed ?

Reply:
just modified the code to use normal event handling registring/unregistring.. 

ill revert back but 

does _sizeChangedDisposable.Disposable = null; 
call dispose()? do you generate code there or something

shouldnt we call 
_sizeChangedDisposable.Dispose();
_sizeChangedDisposable.Disposable = null;

or just
_sizeChangedDisposable.Dispose();

is there a true valid reason of not using normal event registring/unregistring?